### PR TITLE
Added link to all plugins section

### PR DESCRIPTION
**Pull request by timtjtim**

### CODE REVIEW

File: internal_packages/github-contact-card/README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Uses GitHub's public API to look up a GitHub user based on their email address,
 and then displays public repos and their stars.
 
 This example is a good starting point for plugins that want to display data from
-external sources in the sidebar.
+external sources in the sidebar. If you want to see some more advanced plugins, have a look through [all the internal plugins, here on Github](https://github.com/timtjtim/N1/tree/master/internal_packages).
```
Comment (KevinMartin) [inline internal_packages/github-contact-card/README.md]: Did you mean to put `https://github.com/nylas/N1/tree/master/internal_packages` instead?

**Comment by timtjtim** [comment]
@KevinMartin Yes, committed correct link.

**Comment by timtjtim** [comment]
@bengotow Why is it only passing 2 of the checks? I take it that's a current bug with N1 which will appear in all forks / pulls?

**Comment by jackiehluo** [comment]
@timtjtim Yeah, looks like two unrelated tests are failing. No worries!

**Pull request merged**